Gems/Atom/RPI/Code/Source/RPI.Reflect/RPISystemDescriptor.cpp

```diff
@@ -26,34 +26,11 @@ namespace AZ
 
                 serializeContext->Class<RPISystemDescriptor>()
                     ->Version(6) // ATOM-15472
-                    ->Field("RHISystemDescriptor", &RPISystemDescriptor::m_rhiSystemDescriptor)
```

Comment:
This is going to need to update the Version number due to the field being removed, otherwise serialization is going to error about an unknown field.

Reply:
I could. Although, we didn't serialize this data to any data files.